### PR TITLE
Added Passer!

### LINKED ISSUE #130: Make passer
- [ ] make passer subsystem
- [ ] make pass command
- [ ]    -make note go into rollers and move back before shooting
- [ ] add command to joystick

### PR DESCRIPTION
**Pull request by Griffin9881**
### Pull Request

---------------------------------------------

#### Issue Number

Closes #130

### CODE REVIEW

File: src/main/java/frc/robot/commands/ShootPasserCommand.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    @Override
+    public void initialize() {
+        timer.start();
+    }
+
+    @Override
+    public void execute() {
+        if (storage.noteExitingStorage()) {
+            scoring.feedForAmp();
```
Comment (reediculous456) [inline src/main/java/frc/robot/commands/ShootPasserCommand.java:33]: Why not just use feed for passer all the time?

File: src/main/java/frc/robot/constants/StorageConstants.java [hunk 2]
```diff
@@ -3,8 +3,8 @@
 public class StorageConstants {
 
     public static final int transitionMotorPort = 7;
-    public static final int leftStoragePort = 11;
-    public static final int rightStoragePort = 8;
+    public static final int storagePort = 11;
+    // public static final int rightStoragePort = 8;
```
Comment (reediculous456) [inline src/main/java/frc/robot/constants/StorageConstants.java:7]: Please remove, not comment

File: src/main/java/frc/robot/subsystems/Passer.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    public static Passer getInstance() {
+        if (instance == null) {
+            instance = new Passer();
+        }
+        return instance;
+    }
+
+    private Passer() {
+        passerMotor.restoreFactoryDefaults();
+        passerMotor.setIdleMode(IdleMode.kBrake);
```
Comment (reediculous456) [inline src/main/java/frc/robot/subsystems/Passer.java:33]: Probably should be coast like shooter

**Review by reediculous456 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Passer.java (hunk 3 above)
Comment (Griffin9881) [reply src/main/java/frc/robot/subsystems/Passer.java:33]: isn't shooter brake? or is that a different motor than the roller motor?

File: src/main/java/frc/robot/commands/ShootPasserCommand.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    @Override
+    public void initialize() {
+        timer.start();
+    }
+
+    @Override
+    public void execute() {
+        scoring.feedForAmp();
```
Comment (reediculous456) [inline src/main/java/frc/robot/commands/ShootPasserCommand.java:32]: Calling both feed for amp and feed for passer seems redundant here

**Review by reediculous456 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Passer.java (hunk 3 above)
Comment (reediculous456) [reply src/main/java/frc/robot/subsystems/Passer.java:33]: Maybe I’m wrong but I thought it was in coast. If not this is fine

**Pull request merged**